### ISSUE TITLE
Create Documents upload feature 

### ISSUE THREAD

**Issue by DikshaDisciplines** [AJaySi/AI-Writer#62]

**Comment by AJaySi** [comment]
Yup, Alwrity needs a ContentFromDocs features. I will soon take this up.

**Comment by AJaySi** [comment]
@DikshaDisciplines 

We are working on the new user interface(streamlit UI). Alwrity will use files present in a designated folder. It will also work with web urls, domain names, audio and video files. Starting work on this next.

**Comment by AJaySi** [comment]
@DikshaDisciplines 

1). Alwrity is multimodal now. 
2). Blog from youtube, web urls.
3). Blog from local audio file (mp3, mp4, wav).
4). Blog from local image file(png, jpeg, webp).

Working on PDF, txt, docx files.

**Comment by merihakca** [comment]
I just came here to ask the same request of PDF and DOC upload features, hope you guys can accomplish soon !!! Thanks a lot for your amazing techno-art work you create.

**Comment by AJaySi** [comment]
@merihakca @DikshaDisciplines 

Hello,
   Oops, kinda forgot about this ticket. I will take this up in this week and then update you guys.
Regards & Thanks.

**Comment by AJaySi** [comment]
@merihakca @DikshaDisciplines 

I have included attaching a long pdf file to extract insights from it.

Test: Attach a PDF file and simply click 'Write Blog'.
Given a very long document, Alwrity will provide insights with citations.

Presently, works with openai, making changes for Gemini model. I will push the code tomorrow.

**Comment by AJaySi** [comment]
@merihakca @DikshaDisciplines 
I have uploaded the code. Please check & let me know if you would want insights or blog content from it. 
Presently, the text input is ignored, but we can add the text as prompt with the attached document.

Regards.

**Comment by AJaySi** [comment]
Closing it, assuming all good.
